crates/bevy_render/src/camera/projection.rs

```diff
@@ -201,6 +201,36 @@ impl CustomProjection {
     }
 }
 
+// TODO: remove when trait upcasting is stabilized.
```

Comment:
Trait upcasting is merged into Rust, but not *quite* released. See https://github.com/rust-lang/rust/issues/65991/. Does it make sense to wait until April 3rd? https://releases.rs/docs/1.86.0/